lm_eval/models/optimum_lm_genai.py

```diff
@@ -0,0 +1,164 @@
+from importlib.util import find_spec
+from pathlib import Path
+
+from lm_eval.api.registry import register_model
+from lm_eval.models.huggingface import HFLM
+
+import sys
+for path in sys.path:
+    print(path)
+
+from llm_bench.python.llm_bench_utils.ov_utils import create_text_gen_model, build_ov_tokenizer
+from llm_bench.python.llm_bench_utils.model_utils import get_use_case
+
+
+@register_model("optimum-causal")
```

Comment:
Let's call it `openvino-causal` and the class name `OpenVINOCausalLM`

Reply:
Done